GTG/gtk/editor/taskview.py

```diff
@@ -463,6 +463,37 @@ def insert_tag(self, tag):
         line_mark = self.buff.create_mark("lastline", lastline, False)
         self.insert_at_mark(self.buff, line_mark, tag)
 
+    def remove_tag_editor(self, tagname):
+        '''
+        TODO(jakubbrindza): Finish works on this function which will
+        enable removing a tag from a task in task editor.
```

Comment:
Currently you can't remove tag from the toolbar, only to add it. After this big editor PR, I'm expecting that you do N small ones, each addressing specific issue (go to parent popover, tag management, start date, due date)